### PR TITLE
KAA-1649: Owner verifiers - reviewed. [master]

### PR DESCRIPTION
**Pull request by hunch7**

### CODE REVIEW

File: doc/Customization-guide/Customizable-system-components/Owner-verifiers/index.md [hunk 1]
```diff
@@ -10,29 +10,32 @@ sort_idx: 40
...
-2. Implement the owner verifier based on AbstractKaaUserVerifier.
+
+2. Implement the owner verifier based on the [AbstractKaaUserVerifier]({{github_url}}server/common/verifier-shared/src/main/java/org/kaaproject/kaa/server/common/verifier/AbstractKaaUserVerifier.java) class.
+
 3. Develop the owner verifier descriptor.
+
 4. Provision the owner verifier.
 
-We recommend that you use one of the [existing owner verifier implementations]({{github_url}}server/verifiers) as a reference.
+It is recommended that you use one of the existing [existing owner verifier implementations]({{github_url}}server/verifiers) as a reference.
```
Comment (Polinaa) [inline doc/Customization-guide/Customizable-system-components/Owner-verifiers/index.md]: two times _existing_

File: doc/Customization-guide/Customizable-system-components/Owner-verifiers/index.md [hunk 2]
```diff
@@ -54,17 +57,19 @@ A owner verifier configuration schema is an Avro compatible schema that defines
...
 java -jar /path/to/avro-tools-1.7.7.jar compile schema <schema file> <destination>
 ```
 
-Please refer to [Compiling the schema](http://avro.apache.org/docs/current/gettingstartedjava.html#Compiling+the+schema) for more information. It is also possible to integrate the schema compilation with [avro-maven-plugin](http://avro.apache.org/docs/current/gettingstartedjava.html).
+See also [Compiling the schema](http://avro.apache.org/docs/current/gettingstartedjava.html#Compiling+the+schema). You can also integrate the schema compilation using the [avro-maven-plugin](http://avro.apache.org/docs/current/gettingstartedjava.html).
 
 ## Owner verifier implementation
 
-All Kaa owner verifiers extend generic abstract class org.kaaproject.kaa.server.common.verifier.AbstractUserVerifier<T>. The following code example illustrates the implementation of a custom owner verifier.
+All Kaa owner verifiers extend generic abstract class `org.kaaproject.kaa.server.common.verifier.AbstractUserVerifier`.
```
Comment (Polinaa) [inline doc/Customization-guide/Customizable-system-components/Owner-verifiers/index.md]: Abstract**Kaa**UserVerifier

File: doc/Customization-guide/Customizable-system-components/Owner-verifiers/index.md (hunk 1 above)
Comment (hunch7) [reply doc/Customization-guide/Customizable-system-components/Owner-verifiers/index.md]: Fixed.

File: doc/Customization-guide/Customizable-system-components/Owner-verifiers/index.md (hunk 2 above)
Comment (hunch7) [reply doc/Customization-guide/Customizable-system-components/Owner-verifiers/index.md]: Fixed.

**Review by Polinaa — APPROVED** [review]

**Review by vadimol — APPROVED** [review]

**Pull request merged**